liquidator/liquidator.js

```diff
@@ -284,8 +286,26 @@ class Liquidator {
 
       // Send the transaction or report failure.
       let receipt;
+
       try {
-        receipt = await liquidation.send(txnConfig);
+        // Make sure to keep trying with this nonce
+        const nonce = await this.web3.eth.getTransactionCount(this.account);
+
+        // Min Gas Price, with a max gasPrice of x4
+        const minGasPrice = parseInt(this.gasEstimator.getCurrentFastPrice(), 10);
+        const maxGasPrice = 4 * minGasPrice;
```

Comment:
Is this correct? if the price doubles each time, for up to 3 re-tries, then should it not be:
`const maxGasPrice = 2 * 3 * minGasPrice;` as the max increase is doubled, 3 times?

Reply:
@kendricktan?